README.md

```diff
@@ -75,6 +75,7 @@ CREATE TABLE IF NOT EXISTS {your_journal_table_name} (
     created_at BIGINT NOT NULL,
     manifest VARCHAR(500) NOT NULL,
     payload BLOB NOT NULL,
+	tags VARCHAR(2000) NULL,
```

Comment:
I've moved to varchar(100) in other implementations. I think, there's no reason to keep that number that high by default.